app/models/manageiq/providers/redhat/infra_manager/vm.rb

```diff
@@ -6,40 +6,39 @@ class ManageIQ::Providers::Redhat::InfraManager::Vm < ManageIQ::Providers::Ovirt
 
   supports :migrate do
     if blank? || orphaned? || archived?
-      unsupported_reason_add(:migrate, "Migrate operation in not supported.")
-    elsif !ext_management_system.supports?(:migrate)
-      unsupported_reason_add(:migrate, 'RHV API version does not support migrate')
+      "Migrate operation in not supported."
+    else
+      ext_management_system.unsupported_reason(:migrate)
     end
```

Comment:
wasn't sure if we could get away with:

```ruby
supports :migrate do
  unsupported_reason_add(:action) || ext_management_system.unsupported_reason(:migrate)
```

Also thinking the `blank?` wants to be `ext_management_system.blank?` (basically will follow the `if ems ; (:active) || (:migrate) ; else _("no ems")` pattern